sprokit/src/sprokit/pipeline/#pipeline.cxx#

```diff
@@ -1,2440 +0,0 @@
-/*ckwg +29
```

Comment:
Would it be possible to modify the history to remove this file entirely, so it was never present in the first place?

Reply:
I can do that when I merge the branch